backend/prompt_studio/prompt_studio_core_v2/prompt_ide_base_tool.py

```diff
@@ -16,6 +19,13 @@ def __init__(self, log_level: LogLevel = LogLevel.INFO, org_id: str = "") -> Non
         """
         self.log_level = log_level
         self.org_id = org_id
+        self.workflow_filestorage = None
```

Comment:
Hope this line will safe guard us from the exception
PromptIdeBaseTool' object has no attribute 'workflow_filestorage'

Reply:
Yes, it will remain None until the feature flag is enabled.